### ISSUE TITLE
Windows linker error

### ISSUE THREAD

**Issue by Kl4rry** [qnighy/libwebp-sys2-rs#12]
When compiling using `cargo install` on version 0.1.2 the compilation fails on the linker error.
```
error LNK2019: unresolved external symbol VP8LDspInitSSE41 referenced in function VP8LDspInit_body
```
The problem gets even worse when using 0.1.4 the error appears every time the library is comiled.

I belive it happend only when using `cargo install` before because `cargo install` uses `target-cpu=native`.

Compiler versions tested:
```
nightly-x86_64-pc-windows-msvc - rustc 1.63.0-nightly (a6b8c6954 2022-06-03)
1.61-x86_64-pc-windows-msvc - rustc 1.61.0 (fe5b13d68 2022-05-18)
```

**Comment by Kl4rry** [comment]
The 0.1.4 version does not compile at all on x86.

**Comment by nirbheek** [comment]
> `error LNK2019: unresolved external symbol VP8LDspInitSSE41 referenced in function VP8LDspInit_body`

This error happens because `lossless_sse41.c` is not bundled with the library.

**Comment by Randall-Coding** [comment]
HI all.  I'm running into the same issue on Windows.

Is this feature still under development and if not does anyone have a work-around?

Compiler version: 
1.69.0-nightly, nightly-x86_64-pc-windows-msvc, Win10

**Comment by Kl4rry** [comment]
The main maintainer seems to not be responding to issues/PRs however the crates.io is also owned by the rust-bus maintainers. We could try contacting them and see if they can merge and publish the existing PR that fixes this issue.

**Comment by Kl4rry** [comment]
Some symbols still seem to be missing. I still get these linker errors on windows.
```
liblibwebp_sys-691dbd7c751c54e8.rlib(picture_csp_enc.o) : error LNK2019: unresolved external symbol SharpYuvConvert referenced in function ImportYUVAFromRGBA
liblibwebp_sys-691dbd7c751c54e8.rlib(picture_csp_enc.o) : error LNK2019: unresolved external symbol SharpYuvGetConversionMatrix referenced in function ImportYUVAFromRGBA
liblibwebp_sys-691dbd7c751c54e8.rlib(picture_csp_enc.o) : error LNK2019: unresolved external symbol SharpYuvInit referenced in function ImportYUVAFromRGBA
```

**Comment by Randall-Coding** [comment]
> Some symbols still seem to be missing. I still get these linker errors on windows.
> 
> ```
> liblibwebp_sys-691dbd7c751c54e8.rlib(picture_csp_enc.o) : error LNK2019: unresolved external symbol SharpYuvConvert referenced in function ImportYUVAFromRGBA
> liblibwebp_sys-691dbd7c751c54e8.rlib(picture_csp_enc.o) : error LNK2019: unresolved external symbol SharpYuvGetConversionMatrix referenced in function ImportYUVAFromRGBA
> liblibwebp_sys-691dbd7c751c54e8.rlib(picture_csp_enc.o) : error LNK2019: unresolved external symbol SharpYuvInit referenced in function ImportYUVAFromRGBA
> ```

Interesting, I'm not seeing those.  I'm running a Win10 VM.

**Comment by Kl4rry** [comment]
This linking error only appears when encoding not when decoding.

**Comment by Kl4rry** [comment]
I think this is fixed in 8e89b48
